docs/usage/getting-started/private-packages.md

```diff
@@ -455,6 +455,34 @@ verify_ssl = true
 name = "pypi"
 ```
 
+### pip-compile
+
+The pip-compile manager will extract `--index-url` and `--extra-index-url` directives from its input file. These URLs will pull credentials from matching `hostRules` blocks and pass appropriate `--index-url` or `--extra-index-url`
+flags to `pip-compile` with those credentials while generating the output file.
+
```

Comment:
```suggestion
The pip-compile manager extracts `--index-url` and `--extra-index-url` directives from its input file.
These URLs pull credentials from matching `hostRules` blocks and pass appropriate `--index-url` or `--extra-index-url` flags to `pip-compile` with those credentials while generating the output file.
```

Let's start small and follow the one sentence per line convention. I find this paragraph hard to understand, so it needs a full rewrite.